src/adapter/http/src/middleware/authentication_layer.rs

```diff
@@ -115,7 +115,7 @@ where
     }
 
     fn call(&mut self, mut request: http::Request<Body>) -> Self::Future {
-        // Inspired by https://github.com/maxcountryman/axum-login/blob/main/axum-login/src/auth.rs
+        // Inspired by https://github.com/maxcountryman/axum-login/blob/5239b38b2698a3db3f92075b6ad430aea79c215a/axum-login/src/auth.rs
```

Comment:
`axum-login` has been rewritten, so I've updated the commit links from the past

Reply:
We might not care about this comment anyway :-)